app/charts/map/map.tsx

```diff
@@ -134,6 +130,43 @@ let globalScatterplotLayerId = 0;
 
 const FLY_TO_DURATION = 500;
 
+const useViewState = ({
```

Comment:
Can you add a docstring, I think it will help nail down the responsibilities of this hook.

Reply:
Sure!